ql/src/java/org/apache/hadoop/hive/ql/stats/BasicStatsTask.java

```diff
@@ -309,9 +309,6 @@ private int aggregateStats(Hive db, Table tbl) {
         TransactionalStatsProcessor transactionalStatsProcessor = new TransactionalStatsProcessor(db, p);
         transactionalStatsProcessor.process(statsAggregator);
 
-        if (conf.getBoolVar(ConfVars.TEZ_EXEC_SUMMARY)) {
```

Comment:
Isnt something similar happening on the partition level as well?
https://github.com/apache/hive/blob/master/ql/src/java/org/apache/hadoop/hive/ql/stats/BasicStatsTask.java#L378-L381

Also `console` variable seems to wrap the logger (LOG) variable and essentially doing things that can be achieved via the logger object directly. Maybe remove the console variable?

Reply:
Ohh this is an excellent catch @SourabhBadhya

Reply:
Definitely a good catch!
I also take a little time to check `LOG.info` and `console.printInfo`, and it seems that the` else if `is more reasonable.
Thanks.